test/api-integration-test/run_test.sh

```diff
@@ -60,8 +60,11 @@ TEST_DIR=backend/test
 
 echo "Clone ML pipeline code in COMMIT SHA ${COMMIT_SHA}..."
 git clone https://github.com/${GITHUB_REPO} ${BASE_DIR}
-cd ${BASE_DIR}/${TEST_DIR}
-git checkout ${COMMIT_SHA}
+cd ${BASE_DIR}
+git config --local user.name 'K8S Bootstrap'
```

Comment:
do these config needed? i would recommend not adding them if not required, for portability purpose.

Reply:
seems it's required. lgtm